### PR TITLE
Adjust default map/directory blob summary sizes

### PR DESCRIPTION
**Pull request by vladsud**
See discussion and data in https://dev.azure.com/fluidframework/internal/_workitems/edit/3831?src=WorkItemMention&src-action=artifact_link

### CODE REVIEW

**Comment by vladsud** [comment]
@marcmasmsft, can you please share your thoughts on optimal blob size from ODSP perspective?
@yangg-msft - could you please do the same from POV of FRS?

**Comment by Abe27342** [comment]
FYI: bump here is in the wrong package :). This was some WIP support for attribution in SharedMap (where intent was to eventually port changes to real SharedMap). No objections changing it in both places, but I'm guessing you care more about prod codepath.

File: experimental/dds/attributable-map/src/map.ts [hunk 1]
```diff
@@ -287,12 +287,21 @@ export class AttributableMap extends SharedObject<ISharedMapEvents> implements I
...
 		// Should be bigger than MinValueSizeSeparateSnapshotBlob
-		const MaxSnapshotBlobSize = 16 * 1024;
+		// There is no strict requirement for this value, but it should be reasonable.
+		// Reasonably large, such that relative overhead of creating multiple blobs is not too high.
+		// Reasonably small, such that we don't create so large blobs that storage system has to split them.
+		// For example, ODSP stores content in 1Mb Azure blobs. That said, it stores compressed content, so the size of
+		// blobs has only indirect impact on storage size.
+		// Please note that smaller sizes increase the chances of blob reuse across summaries. That said
+		// we have no code on client side to do such dedupping. Service side blob dedupping does not help much (we still transfer bites over wire).
+		const MaxSnapshotBlobSize = 256 * 1024;
```
Comment (tyler-cai-microsoft) [inline experimental/dds/attributable-map/src/map.ts:304]: Doing a quick mental check here, both old and new clients shouldn't need to worry about back compat here right? I'm assuming they both should be able to open these blobs no problem.

**Review by tyler-cai-microsoft — APPROVED** [review]
LGTM

File: experimental/dds/attributable-map/src/map.ts (hunk 1 above)
Comment (vladsud) [reply experimental/dds/attributable-map/src/map.ts:304]: Correct, they reading part does not care how big are the blobs, only writing part is affected

**Comment by vladsud** [comment]
FYI, storing for historic records, FRS take on optimal size:
> ... in some Azure Blob tests we did a while back, we saw more failures when the blob size was greater than 10MB
> ... Overall, I'd say a size between 1MB and 5MB seems to be a sweet spot.

I'm not ready to go that far yet, and stats (in the ADO item, even though it's ODSP stats) suggests we do not generate such big blobs. So, let's start with a bit smaller value for now - it's already a huge change and increment even further if there is enough data it's warranted.

**Pull request merged**